example/src/screens/timelineCalendar.tsx

```diff
@@ -9,95 +9,98 @@ import {
   CalendarUtils
 } from 'react-native-calendars';
 import _ from 'lodash';
+import XDate from 'xdate';
```

Comment:
The screens should not use XDate

Reply:
Technically I just used it for formatting the strings, I didn't pass it to the component. 
Anyway, fair enough, I replaced it with `Date`